### DISCUSSION TITLE
Duplicate Instances and Port Issues with SLZB-06

### DISCUSSION THREAD

**Discussion by timwagg** [Koenkk/zigbee2mqtt#22382] [Devices]
Hi,

I've recently purchases a SLZB-06 to extend my zigbee coverage - it's connected via ethernet and I am able to access the web interface but I cannot get my Z2M instance to connect. I already have a Sonoff Zigbee USB plugged directly into my HA device and I have a Z2M instance running fine on port 8485. 

From what I have read and watched, in order to have my Sonoff router and my SLZB running at the same time I need to run two instances which I have successfully done but according to (https://youtu.be/QOx733CU1p8) you need to change the port for each instance but when I connect it fails and kicks out a 502 error.

I am running a cloudflared tunnel for remote access but from what I can gather this isn't the issue as when I connect locally I get another 502 error without the cloudflared branding.

Can anyone help? I'm stumped. Tried many different ports and none work. And when I try to change the Sonoff port away from the default it fails.

Thanks in advance!

`[21:40:50] INFO: Preparing to start...
[21:40:50] INFO: Socat not enabled
[21:40:52] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:debug 2024-05-01 21:40:59: Loaded state from file /config/zigbee2mqtt/state.json
Zigbee2MQTT:info  2024-05-01 21:40:59: Logging to console and directory: '/config/zigbee2mqtt/log/2024-05-01.21-40-58' filename: log.txt
Zigbee2MQTT:debug 2024-05-01 21:40:59: Removing old log directory '/config/zigbee2mqtt/log/2024-05-01.21-05-58'
Zigbee2MQTT:info  2024-05-01 21:40:59: Starting Zigbee2MQTT version 1.36.1 (commit #unknown)
Zigbee2MQTT:info  2024-05-01 21:40:59: Starting zigbee-herdsman (0.40.3)
Zigbee2MQTT:debug 2024-05-01 21:40:59: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"path":"/dev/ttyUSB0"}}'
Zigbee2MQTT:error 2024-05-01 21:40:59: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2024-05-01 21:40:59: Failed to start zigbee
Zigbee2MQTT:error 2024-05-01 21:40:59: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2024-05-01 21:40:59: Exiting...
Zigbee2MQTT:error 2024-05-01 21:40:59: Error: Error Resource temporarily unavailable Cannot lock port`

**Comment by francisp2** [comment]
What 'port' did you change ? 
For serial port, your Sonoff USB use /dev/serial/by-id/...
For the SLZB-06 it should be tcp://ip:port

Dont forget to change the MQTT ,topic so it is different for each instance.

**Comment by timwagg** [comment]
I changed the port at the bottom of the Home Assistant add on settings in the UI, under Network in the zigbee2mqtt config options.

Serial is set as you suggested for both devices and the base is also different.

**Reply by timwagg** [reply]
oddly this is now working, i updated Z2M and after initially failing to connect it now does. I also reinstalled the instance so maybe that helped? Sorry I can't share any more detail than that.
